config/defaults.mk

```diff
@@ -353,6 +354,11 @@ ifeq ($(SLEPC_FOUND),YES)
       $(subst $(CXX_XLINKER),$(XLINKER),$(SLEPC_DEP))
 endif
 
+ifeq ($(MFEM_USE_MOONOLITH),YES)
+  include $(MOONOLITH_DIR)/config/moonolith-config.makefile
```

Comment:
I cloned and installed moonolith, but I don't see this folder/file anywhere. Is this assuming something different than moonolith's `master` branch? It's not clear how to set `MOONOLITH_DIR`.

Reply:
I have added more specific instructions. In short non-Cmake users need to install monolith with make install_all which will generate the make config file.